contracts/scope/artifacts/checksums_intermediate.txt

```diff
@@ -0,0 +1 @@
+3be93227e90d8a81d1a575dd6838136276e33d8e93cc595acc68b64d5f35893b  ./target/wasm32-unknown-unknown/release/scope.wasm
```

Comment:
havent seen these before, should these exist? just did a `make all`

Reply:
Hmm, I wonder if that should be added to the .gitignore?  Doesn't seem like we would want to commit those?

Reply:
yeah i can take them out; just surprised they were generated and not added to gitignore before?

Reply:
its also weird that it wrote to these new files instead of the old checksums.txt - maybe a rust version thing?